setup.py

```diff
@@ -48,16 +48,29 @@ def extra(self):
     except ImportError:
         pass
     else:
+        # For cython test coverage:
+        # Uncomment the compiler_directives line below
+        # Run: python setup.py build_ext --force --inplace --define CYTHON_TRACE
```

Comment:
You can get test coverage reports for the cythonized code by following these instructions. Based on https://medium.com/@dfdeshom/better-test-coverage-workflow-for-cython-modules-631615eb197a

There may be a good way to automate this as part of a test build but that's beyond my knowledge.

Reply:
I'll do it after we merge this.